client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -0,0 +1,465 @@
+import styled from '@emotion/styled';
+import { translate, useTranslate } from 'i18n-calypso';
+import { ChangeEvent, useCallback, useState } from 'react';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormRadio from 'calypso/components/forms/form-radio';
+import FormInput from 'calypso/components/forms/form-text-input';
+import { useSelector } from 'calypso/state';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { ConfirmationModal } from '../confirmation-modal';
+import SyncOptionsPanel, { CheckboxOptionItem } from '../sync-options-panel';
+import { useCheckSyncStatus } from '../use-site-sync-status';
+import { StagingSiteSyncLoadingBarCardContent } from './staging-site-sync-loading-bar-card-content';
+
+const synchronizationOptions: CheckboxOptionItem[] = [
+	{
+		name: 'sqls',
+		label: 'Site Database (SQL)',
+		subTitle: translate(
+			'Overwrite the database, including any posts, pages, products, or orders with staging data.'
+		),
+		checked: false,
+		isDangerous: true,
+	},
+	{
+		name: 'themes',
+		label: translate( 'Themes' ),
+		checked: false,
+		isDangerous: false,
+	},
+	{
+		name: 'plugins',
+		label: translate( 'Plugins' ),
+		checked: false,
+		isDangerous: false,
+	},
+	{
+		name: 'uploads',
+		label: translate( 'Media Uploads' ),
```

Comment:
It seems that media uploads require database synchronization, too. In the other case, by default, they won't appear on the destination site.

Would it make sense to clarify this here? From [docs](https://wordpress.com/support/restore/):

> Contents of the media library. You must also select ‘Site Database‘ for synchronized media uploads to appear.

Reply:
What if we keep the same wording as in Jetpack UI? Here is what will look like with that change in place.
![2023-10-05T14:35:19,403459067+03:00](https://github.com/Automattic/wp-calypso/assets/497103/1b4ef943-39de-4208-babf-e07909e47474)